docs/controls/Bubble.md

```diff
@@ -0,0 +1,365 @@
+# Bubble
+
+A native bubble graph using D3 based on standard design patterns.
+
+-   [Bubble](#bubble)
+    -   [Usage](#usage)
+        -   [Structure](#structure)
+            -   [Weight Based Bubble](#weight-based-bubble)
+            -   [Color Based Bubble](#color-based-bubble)
+            -   [Weight Color Based Bubble](#weight-color-based-bubble)
+            -   [Custom Size Based Bubble](#custom-size-based-bubble)
+    -   [JSON Properties](#json-properties)
+        -   [Root](#root)
+        -   [Data](#data)
+            -   [Required](#required)
+            -   [Optional](#optional)
+        -   [Values](#values)
+            -   [Required](#required-2)
+            -   [Optional](#optional-2)
+        -   [Regions](#regions)
+            -   [Required](#required-3)
+            -   [Optional](#optional-3)
+        -   [Constraints](#constraints)
+
+## Usage
+
+### Structure
+
+You will **not** need all the properties in the example below.
+Check out _optional_/_required_ properties explained in the [JSON Properties](#json-properties) section.
+
+```javascript
+var root = {
+    bindTo: "#root",
+    axis: {
+        x: {
+            type: Carbon.helpers.AXIS_TYPE.TIME_SERIES,
+            label: "Some X Label",
+            lowerLimit: "2016-01-01T12:00:00Z",
+            upperLimit: "2017-01-01T12:00:00Z"
+        },
+        y: {
+            label: "Some Y Label",
+            lowerLimit: 0,
+            upperLimit: 20
+        },
+        y2: {
+            show: false,
+            label: "Some Y2 Label",
+            lowerLimit: 0,
+            upperLimit: 250
+        }
+    },
+    dateline: [
+        {
+            showDatelineIndicator: true,
+            label: {
+                display: "Release A"
+            },
+            color: Carbon.helpers.COLORS.GREEN,
+            shape: Carbon.helpers.SHAPES.DARK.TRIANGLE,
+            onClick: (onCloseCB, payload) => {
+                // onCloseCB needs to called by the consumer after popup is closed;
+                // Payload is the dateline input object
+            },
+            value: new Date(2016, 5, 1).toISOString()
+        }
+    ],
+    clickPassThrough: {
+        datelines: false
+    },
+    showLabel: true,
```

Comment:
I would remove everything other than what's required for all these graphs. I am not saying what you did here is wrong at all, this is great but since there are different examples we would want the user to provide only information they absolutely need to render a bubble graph. 
For instance, dateline, showLabel etc. flags, onClick, regions can all be removed. 
Have 3-4 bubbles as values and not more.

Reply:
I will do the change.